numba/tests/test_jitclasses.py

```diff
@@ -11,6 +11,13 @@
 from numba import unittest_support as unittest
 from numba import jitclass
 from .support import TestCase, MemoryLeakMixin, tag
+from numba.jitclass import _box
+from numba.runtime.nrt import MemInfo
+
+
+def _get_meminfo(box):
+    ptr = _box.box_get_meminfoptr(box)
+    return MemInfo(ptr, owned=0)  # borrow ownership
```

Comment:
Yes, but I don't want to change the tests too much initially (as the tests check for whether the refcount matches the expected value).  It's probably ok now to take the `acquire()` route and remove the `owned` attr.